compiler-rt/lib/builtins/clzdi2.c

```diff
@@ -14,12 +14,12 @@
 
 // Returns: the number of leading 0-bits
 
-#if !defined(__clang__) &&                                                     \
-    ((defined(__sparc__) && defined(__arch64__)) || defined(__mips64) ||       \
+#if ((defined(__sparc__) && defined(__arch64__)) || defined(__mips64) ||       \
```

Comment:
This doesn't make sense ... `defined(__sparc__) && defined(__aarch64__)`. I think that is supposed to be `||`.